src/component/button/button.scss

```diff
@@ -56,6 +56,26 @@
       }
     }
 
+    &.catalog {
+      border: 1px solid themed('secondary-color');
+      color: themed('primary-text');
+      border-radius: 12px;
+      background: themed('secondary-color');
+      box-shadow: 0 4px 12px rgba(255, 149, 0, 0.6);
```

Comment:
you can add this color `rgba(255, 149, 0, 0.6)` to the palette

Reply:
I found out we can just add the primary color in the rgb function so I didn't need to add another color